vault_s2i/Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM vault:1.1.2
+FROM python:3-stretch
 
 COPY . /
-
 ARG VAULT_GUEST_PORT
-RUN apk add --no-cache python3 && \
-    pip3 install -r /requirements.txt && \
+RUN pip install -r /requirements.txt && \
     export VAULT_GUEST_PORT=$VAULT_GUEST_PORT && \
-    j2 /config.hcl.j2 -o /vault/config/config.hcl
+    j2 /config.hcl.j2 -o /config.hcl
+
+
+FROM vault:1.1.2
+
+COPY --from=0 /config.hcl /vault/config/config.hcl
```

Comment:
Can you please elaborate on the change?
Why do we even need python inside the vault image?

Reply:
We need pip to install jinja2 which process config.hcl.j2 file